### DISCUSSION TITLE
ECHIPA STRAINII

### DISCUSSION THREAD

**Discussion by AdrianaCionca** [peviitor-ro/community#125] [TEAMs]
Iurie Chigai
role: scraper developer
discord: Iurie

**Comment by AdrianaCionca** [comment]
Claudia Elena Rusu
role: Team Lead and Manual Tester
discord: Claudia

**Comment by AdrianaCionca** [comment]
Mihaela Joita
role: Manual Tester
discord: Mihaela

**Comment by Catalinay** [comment]
Catalina Ion
role: Manual Tester
discord: Cat
